lib/aruba/api.rb

```diff
@@ -677,8 +681,8 @@ def check_directory_presence(paths, expect_presence)
     def prep_for_fs_check(&block)
       warn('The use of "prep_for_fs_check" is deprecated. It will be removed soon.')
 
-      stop_processes!
-      in_current_directory { block.call }
+      process_monitor.stop_processes!
```

Comment:
For now it's named `process_monitor`. This will be renamed to `command_monitor` or something similar in #259.